React/Views/ScrollView/RCTScrollView.m

```diff
@@ -817,12 +821,36 @@ - (void)scrollViewDidEndZooming:(UIScrollView *)scrollView withView:(UIView *)vi
   RCT_FORWARD_SCROLL_EVENT(scrollViewDidEndZooming : scrollView withView : view atScale : scale);
 }
 
+- (void)didMoveToWindow
+{
+  [super didMoveToWindow];
+  if (self.window == nil) {
+    if(_isScrollBeginFired == YES && _isScrollEndFired == NO) {
+      // Fire the end deceleration event
+      RCT_SEND_SCROLL_EVENT(onMomentumScrollEnd, nil);
+      RCT_FORWARD_SCROLL_EVENT(scrollViewDidEndDecelerating : _scrollView);
+      _isScrollEndFired = YES;
+      _isScrollBeginFired = NO;
+    }
+  }
```

Comment:
we are executing this code when the scroll view has already been dismissed (the window is nil).
Do we actually need to emit 
```
RCT_SEND_SCROLL_EVENT(onMomentumScrollEnd, nil);
RCT_FORWARD_SCROLL_EVENT(scrollViewDidEndDecelerating : _scrollView);
```
My fear is that the receiver for that is not on the screen anymore and thus it may crash JS/the app if it has been deallocated in native.